common4j/src/main/com/microsoft/identity/common/java/util/UrlUtil.java

```diff
@@ -43,15 +43,25 @@
 public class UrlUtil {
     private static final String TAG = UrlUtil.class.getSimpleName();
 
+    public static URL appendPathToURL(@NonNull final URL urlToAppend,
+                                      @Nullable final String pathString)
+            throws URISyntaxException, MalformedURLException {
+        return appendPathToURL(urlToAppend, pathString, false, null);
+    }
+
     /**
      * Append a given path string to a URL.
      *
      * @param urlToAppend URL to be appended to.
      * @param pathString  a string containing path segments to be appended to the URL.
+     * @param removeEmptySegments flag to denote if empty path segments should be removed
+     * @param queryParam Query parameters to add at the end of url
      * @return appended URL
      */
     public static URL appendPathToURL(@NonNull final URL urlToAppend,
-                                      @Nullable final String pathString)
+                                      @Nullable final String pathString,
```

Comment:
Left comment about this being Nullable above

Reply:
I have removed the nullable